src/components/Player/Header/PlayerHeader.jsx

```diff
@@ -107,78 +150,62 @@ const getRegistrationBadge = registered => registered && (
   />
 );
 
-const MedalStyled = styled.div`
-  .rankMedal {
-    position: relative;
-    height: 37px;
-    width:37px;
-    top: 3px;
-    margin-right: 5px;
-    text-align: center;
-    &[data-hint-position="top"] {
-      &::after {
-        margin-bottom: 3px;
-        margin-left: -37px;
-      }
-  
-      &::before {
-        top: -3px;
-        margin-left: 13px;
-      }
-    }
-    &-icon {
-      top: 0px;
-      height: 100%;
-      width: 100%;
-    }
-    &-star {
-      position: absolute;
-      left: 0px;
-      height: 100%;
-      width: 100%;
-    }
-  }
-`;
-
-const getRankTierMedal = (rankTier) => {
+const getRankTierMedal = (rankTier, leaderboardRank) => {
   let medalElement = null;
   const imgDescription = rankTierToString(rankTier);
   if (rankTier && rankTier > 9) {
-    const intRankTier = parseInt(rankTier, 10);
-    const iconPath = `/assets/images/dota2/rank_icons/rank_icon_${parseInt(intRankTier / 10, 10)}.png`;
-    const star = parseInt(intRankTier % 10, 10);
-    let correctStar = 0;
-    if (star <= 0) {
-      correctStar = 0;
-    } else if (star >= 5) {
-      correctStar = 5;
+    if (rankTier === 76 && leaderboardRank) {
+      let iconPath = '/assets/images/dota2/rank_icons/rank_icon_7a.png'; // Divine Elite
```

Comment:
curious, is "Divine Elite" an official Valve specification?

Reply:
![image](https://user-images.githubusercontent.com/17739114/33706960-ce39483e-db70-11e7-8039-e158e33bf9ff.png)
I found this image in a dota2 website of our country, not sure about it. Maybe in the US, you can easily find it.

Reply:
do you have a link to that site?

Reply:
the origin link which I found is [this](https://weibo.com/3992841761/Fyl23p21n?type=comment#_rnd1512637808952)
Weibo is our country's twitter, but I still don't find a official annoucement about ranked medals

Reply:
And according to some comments, OMG is using the turbo mode